### PR TITLE
[TPA-2022] Creating 2022 Event for Tampa Bay

### PR DESCRIPTION
**Pull request by jmgress**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

### CODE REVIEW

**Comment by mattstratton** [comment]
The deploy preview doesn't work - I'll have to check but something didn't go quite right with just renaming all the files. I'll investigate it tomorrow on the computer.

**Comment by jmgress** [comment]
Thanks, @mattstratton .  If there is something I can do please let me know.

File: data/events/2022-tampa.yml [hunk 1]
```diff
@@ -8,12 +8,12 @@ masthead_background: "logo-square.jpg"
...
 # All dates are in unquoted YYYY-MM-DD or YYYY-MM-DDTHH:MM:SS+TZ:TZ, like this:   variable: 2016-01-05T23:59:59+02:00
-startdate: "" # The start date of your event. Leave blank if you don't have a venue reserved yet.
-enddate: "" # The end date of your event. Leave blank if you don't have a venue reserved yet.
+startdate: "2022-05-27" # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: "2022-05-27" # The end date of your event. Leave blank if you don't have a venue reserved yet.
 
 # Leave CFP dates blank if you don't know yet, or set all three at once.
 
-cfp_date_start: "" # start accepting talk proposals.
+cfp_date_start: "12-30-2021" # start accepting talk proposals.
```
Comment (mattstratton) [inline data/events/2022-tampa.yml:16]: Incorrect formatting

File: data/events/2022-tampa.yml [hunk 2]
```diff
@@ -8,12 +8,12 @@ masthead_background: "logo-square.jpg"
...
 cfp_date_end: "" # close your call for proposals.
```
Comment (mattstratton) [inline data/events/2022-tampa.yml:17]: You cannot set cfp open without setting close and announce as well

**Pull request merged**